### ISSUE TITLE
Ability to disable Crop inside square corner dragging 

### ISSUE THREAD

**Issue by iampopal** [lykhonis/image_crop#72]
I have a case where not need outside corners, like instagram images cropping 
Need ability to Crop box corners dragging

**Comment by Shiba-Kar** [comment]
yess same here

**Comment by MohsenHaydari** [comment]
yes, but for now you can comment on lines 565 to 576 in` crop.dart` file

**Comment by CodingWithNobody** [comment]
Check this https://www.youtube.com/watch?v=1aRwxnX5nms
